cases/icon-art-oem-test/config.py

```diff
@@ -1,18 +1,22 @@
 import os
 """
-Configuration file for the 'icon-art-test' case with ICON-ART
+Configuration file for the 'icon-art-oem-test' case with ICON-ART
 """
 
 # GENERAL SETTINGS ===========================================================
 user = os.environ['USER']
-target = 'icon-art-oem'
-restart_step = 24  # hours
-
+if os.path.exists(os.environ['HOME'] + '/.acct'):
```

Comment:
Is there a reason this case isn't part of the Jenkins test cases?

Reply:
They require a custom version of icon(-art). If @micstein89 send me instructions, which version to use, I can integrate it into the Jenkins testing.

Reply:
Any updates here? If it requires more work, we can integrate it in a separate PR.